### PR TITLE
Fix misspellings throughout docs

### PR DESCRIPTION
**Pull request by clay-aptos**
### Description

Used aspell CLI to find issues

### Test Plan

Manually fixed docs

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/4592)

### CODE REVIEW

File: developer-docs-site/docs/nodes/ait/node-requirements.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ Example machine types on various clouds:
 
 ### Implications on hardware requirements
 
-The amount of data stored by the Aptos Blockhain depends on the ledger history (the number of transactions) of the blockchain and the number of on-chain states (e.g., accounts and resources). These values depend on several factors, including: the age of the blockchain, the average transaction rate, and the configuration of the ledger pruner.
+The amount of data stored by the Aptos Blockchain depends on the ledger history (the number of transactions) of the blockchain and the number of on-chain states (e.g., accounts and resources). These values depend on several factors, including: the age of the blockchain, the average transaction rate, and the configuration of the ledger pruner.
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/nodes/ait/node-requirements.md:46]: We are no longer title casing this word. Should be "Aptos blockchain". See an initial version of Aptos terms Notion doc: https://www.notion.so/aptoslabs/Aptos-Words-and-Phrases-6b69019dbba24fbdac37f55824dc1639

File: developer-docs-site/docs/guides/building-your-own-wallet.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ import TabItem from '@theme/TabItem';
 
 In order to allow for Aptos' wallet interoperability, the following is required:
 1. Mnemonics - a set of words that can derive account private keys
-2. Dapp API - entry points into the wallet to support access to identity managed by the wallet
+2. dApp API - entry points into the wallet to support access to identity managed by the wallet
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/guides/building-your-own-wallet.md:12]: Just a note for now. I am not too keen on spelling it like "dApp" but it's not a major thing. Just something for us to talk about later on. In this PR this is good.

**Review by rajkaramchedu — APPROVED** [review]
Love it! Made one or two suggestions. Approving. Thanks!

**Review by rajkaramchedu — APPROVED** [review]
See my comments. Approving.

File: developer-docs-site/docs/nodes/ait/node-requirements.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/nodes/ait/node-requirements.md]: Thanks, Raj! This is now fixed. Please note, this was already in caps and merely misspelled.

File: developer-docs-site/docs/guides/building-your-own-wallet.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/guides/building-your-own-wallet.md:12]: Ah, thank you! I was standardizing with the existing:
https://aptos.dev/guides/building-wallet-extension/#step-3-dapp-integration

Please just let me know which way to go forward. Perhaps we can add the desired form to the aforementioned Aptos Words and Phrases list. Thanks, Raj!

Comment (rajkaramchedu) [reply developer-docs-site/docs/guides/building-your-own-wallet.md:12]: For now we can just park this issue, I think. No biggie for now.

Comment (gregnazario) [reply developer-docs-site/docs/guides/building-your-own-wallet.md:12]: I know you're not keep on `dApp` Raj, but I feel like `dApp` is pretty common in the space

File: developer-docs-site/docs/nodes/validator-node/using-gcp.md [hunk 3]
```diff
@@ -193,7 +193,7 @@ This will download all the Terraform dependencies for you, in the `.terraform` f
 15. To summarize, in your working directory you should have a list of files:
     - `main.tf`: The Terraform files to install the `aptos-node` module (from steps 3 and 4).
     - `keys` folder, which includes:
-      - `public-keys.yaml`: Publick keys for the owner account, consensus, networking (from step 10).
+      - `public-keys.yaml`: Public keys for the owner account, consensus, networking (from step 10).
```
Comment (gregnazario) [inline developer-docs-site/docs/nodes/validator-node/using-gcp.md:196]: lol this was copied over every spot

**Review by gregnazario — APPROVED** [review]

**Pull request merged**